DEV-3/DEV-3/NumericTranslator.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+
+
+namespace DEV_3
+{
+    // <summery>
+    // NumericTranslator
+    // translates a numeric from one number system
+    // to another
+    // </summery>
+
+    class NumericTranslator
+    {
+        // numeric after translation into new number system
+        private string numAfterTranslate = "";
+
+        // <summery>
+        // Method TranslateInto
+        // function for translate a numeric from one number system
+        // to another
+        // </summery>
+        // param string _number - is a numeric, which we need to translate
+        // param string _basis - is a base of new number system, 
+        // into which we will translate number _number
+        public string TranslateInto (string _number, string _basis)
+        {
+            if (_number == "0") throw new Exception("Please, enter a number.");
+
+            int number = Convert.ToInt32(_number);
+            int basis = Convert.ToInt32(_basis);
+
+            // by the condition of the task base can be from 2 - 20
```

Comment:
Check validity of number and basis outside of TranslateInto method, as separate method or object